src/utils/ERC1238Approval.ts

```diff
@@ -0,0 +1,96 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { BigNumberish, ethers, Signature } from "ethers";
+
+// Must match the domain of ERC1238Approval.sol
+const getDomain = (chainId: number, verifyingContract: string) => ({
+  name: "ERC1238 Mint Approval",
+  version: "1",
+  chainId,
+  verifyingContract,
+});
+
+const MintApprovaltypes = {
+  MintApproval: [
+    { name: "recipient", type: "address" },
+    { name: "id", type: "uint256" },
+    { name: "amount", type: "uint256" },
+  ],
+};
+const MintBatchApprovalTypes = {
+  MintBatchApproval: [
+    { name: "recipient", type: "address" },
+    { name: "ids", type: "uint256[]" },
+    { name: "amounts", type: "uint256[]" },
+  ],
+};
+
+// TODO: These types should come from typechain but are not exported for some reason
+type MintApprovalStruct = {
```

Comment:
Odd? Mine normally export. I wonder what the problems are. Check your `src/types/index.ts` file.